src/server/system_services/bucket_server.js

```diff
@@ -1102,9 +1102,12 @@ function get_bucket_info({
     };
     const metrics = _calc_metrics({ bucket, nodes_aggregate_pool, hosts_aggregate_pool, tiering_pools_status, info });
 
+    const bucket_last_update = _.get(bucket, 'storage_stats.last_update');
+    const system_last_update = _.get(bucket, 'system.global.last_update');
+    const last_update = system_last_update > bucket_last_update ? system_last_update : bucket_last_update;
```

Comment:
why not set `last_update` inside `make_changes` instead of passing it every time?